src/Modules/CalcOffence.lua

```diff
@@ -955,6 +955,15 @@ function calcs.offence(env, actor, activeSkill)
 			local projMore = skillModList:More(skillCfg, "ProjectileCount")
 			output.BounceCount = m_floor(projBase * projMore)
 		end
+		if skillModList:Flag(skillCfg, "CannotSplit") then
+			output.SplitCountString = "Cannot split"
+		else
```

Comment:
Add the check for Rain type skills here

Reply:
not actualy the fix, as the arrow nova support adds the rain tag and you can still split from those, also misses the concoction skills